force-app/main/default/classes/OpportunityTriggerHandler.cls

```diff
@@ -0,0 +1,105 @@
+public class OpportunityTriggerHandler implements ITriggerHandler {
+	private static Boolean triggerDisabled = false;
```

Comment:
make it not static variable for all classes as we're using it only inside specific handler, however methods isTriggerEnabled() helps to take right value